packages/tenant-process/src/utilities/errorMappers.ts

```diff
@@ -215,8 +220,9 @@ export const m2mRevokeCertifiedAttributeErrorMapper = (
     .with(
       "tenantNotFound",
       "tenantNotFoundByExternalId",
+      "attributeNotFound",
+      "attributeNotFoundInTenant",
```

Comment:
I think that this was correctly a `400` before, since, in this case, it's trying to revoke an attribute that the tenant doesn't have

Reply:
You're right! Fixed 👍  
I also added `"attributeNotFound"` in the same `400` mapping because otherwise it was not caught by any mapper, I think also in that case a bad request makes sense (still trying to revoke an attribute that the tenant doesn't have, no matter if the attribute exists or not in the registry).